docs/development/feature.md

```diff
@@ -787,6 +791,10 @@ export default register;
 
 Next, open your `clutch.config.js` file and add the amiibo configuration highlighted below:
 
+:::info
+When adding a feature directly in the clutch project, the configuration file will be located in `frontend/packages/app/src/clutch.config.js`
```

Comment:
Same comment above applies here

Reply:
This case is more generic I think, since the org scope is not used. It was not immediately clear when following the guide of what file should be changed if a custom gateway was not used. But like the comment above, maybe there is another way of identifying the config file without adding here the specific route?